ui-grid/ui-grid.d.ts

```diff
@@ -1064,6 +1074,15 @@ declare module uiGrid {
         (scrollEvent: JQueryMouseEventObject): void;
     }
 
+    export interface sortChangedHandler {
+        /**
+         * Sort change event callback
+         * @param {IGridInstance} grid instance
+         * @param {IGridColumn} array of gridColumns that have sorting on them, sorted in priority order
+         */
+        (grid: IGridInstance, columns: IGridColumn[]): void;
```

Comment:
Oh, well, that's because I deleted my repository to start anew on another PR. I guess there's no way of doing it the "normal way" now?